scripts/devtool.py

```diff
@@ -182,12 +182,131 @@ def add(file_path, brief):
     with open(file_path, 'w') as f:
       f.write(rendered)
     
-    click.echo(f'✅ File {file_path} created successfully.')
+    click.echo(f'✅ File {os.path.relpath(file_path)} created successfully.')
   else:
     raise click.ClickException(f'File {file_path} already exists.')
 
+def save_with_file_dialog():
+  import tkinter as tk
+  from tkinter import filedialog
+  import os
+
+  dialogTitle = "DevTool - New file from template"
+  parentDir = get_project_root()
+  root = tk.Tk()
+  root.withdraw()
+
+  filePath = filedialog.asksaveasfilename(title=dialogTitle, initialdir=parentDir, defaultextension='*.*')
```

Comment:
we could prompt from the cli, right?
What would you think is the best approach?